warehouse/warehouse.go

```diff
@@ -177,7 +176,7 @@ func (wh *HandleT) backendConfigSubscriber() {
 			if len(source.Destinations) > 0 {
 				for _, destination := range source.Destinations {
 					if destination.DestinationDefinition.Name == wh.destType {
-						namespace := getNamespaceFromDestinationConfig(destination.Config, source.Name, wh.destType)
+						namespace := getNamespaceFromDestinationConfig(destination.Config, source, destination, wh.destType)
```

Comment:
This function actually checks namespace from 
1. destinationConfig
2. warehouseSchemas
3. sourceName

But the name says it is from destinationConfig. Make it generic like getNamespace()

We don't need to pass destination.Config and destination. Only destination is enough.

Reply:
We have 2 db handles, 
1. global to warehouse module 
2. warehouse-handle specific

Since this function is already warehouse-handle scoped, let's not use the global db handle internally.
Change this function to wh.getNamespace() ?

Reply:
changed